### DISCUSSION TITLE
Does it supports node >=20?

### DISCUSSION THREAD

**Discussion by mahendra-shah** [Stun3R/strapi-sdk-js#200] [Q&A] [answered]
I am currently working on a project that utilizes `strapi-sdk-js: v2.2.0` with Node.js version 14.x.x. There is a requirement to upgrade the Node.js version to v20. However, I am unable to find any information regarding the supported Node.js engine for this SDK.

Could someone please assist me in determining whether **strapi-sdk-js** supports Node.js v20 or if there are any specific requirements I should be aware of?

I appreciate your help.

**Comment by Stun3R** [comment, marked as answer]
It definitly supports Node.js version >=16.0
